spec/beaker/dsl/helpers/puppet_helpers_spec.rb

```diff
@@ -581,9 +581,9 @@ def stub_post_setup
         let(:is_pe) { true }
         let(:service_restart) { true }
 
-        it 'bounces puppet twice' do
+        it 'starts puppet once, stops it twice' do
           subject.with_puppet_running_on(host, {})
-          expect(host).to execute_commands_matching(/puppet resource service #{host['puppetservice']}.*ensure=running/).exactly(2).times
+          expect(host).to execute_commands_matching(/puppet resource service #{host['puppetservice']}.*ensure=running/).once
```

Comment:
I believe that changing this unit test means you are breaking API compatibility with previous versions of Beaker--so is this something that should wait for the next major version of Beaker?